questions/queries.py

```diff
@@ -20,9 +20,16 @@ class QuestionQuery(graphene.ObjectType):
 
     question = graphene.Field(QuestionType, id=graphene.UUID())
     questions = graphene.List(QuestionType)
+    my_questions = graphene.List(QuestionType)
```

Comment:
I think we should remove `questions` and rename `my_questions` to `questions`

Reply:
I will do this in the next PR